membership-attribute-service/app/filters/LogRequestsFilter.scala

```diff
@@ -1,28 +0,0 @@
-package filters
-
-import play.api.Logger
-import play.api.mvc._
-import play.api.libs.concurrent.Execution.Implicits.defaultContext
-import scala.concurrent.Future
-import Headers._
-
-/** Logs all requests, the time the app took to respond, and the status of the response */
-object LogRequestsFilter extends Filter {
```

Comment:
Was this unused? Presumably we record all of this information in the Fastly logs anyway...

Reply:
Yes, we ran a full-text search of this object in the codebase and we didn't find it.

Reply:
👍